### ISSUE TITLE
Support for writing the flat ods format

### ISSUE THREAD

**Issue by mayeulk** [ropensci/readODS#103]
It would be great to be able to write in LibreOffice's flat ods (.fods) format.
LibreOffice document are (mostly) zip groups of xml files, which do not work well with git (small changes in the raw data create large changes in the final file) and diff, and are not human readable. To address this, LO created a flat format, .fods for .ods file. 
See https://wiki.documentfoundation.org/Libreoffice_and_subversion
This is similar to writing the files in an archive, without compression (plain text).
See: 
https://superuser.com/questions/1264379/libreoffice-how-to-save-in-git-friendly-format

https://github.com/costerwi/rezip

https://github.com/callegar/Rezip

**Comment by mayeulk** [comment]
Currently, my workaround on linux is, after writing the .ods, do the following in bash shell:

soffice --headless --convert-to fods myfile.ods

**Comment by mayeulk** [comment]
Currently, my workaround on linux is, after writing the .ods, do the following in bash shell:

`soffice --headless --convert-to fods myfile.ods`

Possibly, playing with
`flags = "-r9X"`
in writeODS.R would do?

**Comment by chainsawriot** [comment]
@mayeulk Thank you for suggesting this. I will add this to v2.

**Comment by chainsawriot** [comment]
Fixed by #110

**Comment by chainsawriot** [comment]
@mayeulk Sorry for taking it this long. Preliminary support for fods is now available. Please install the development version.

```r
remotes::install_github("ropensci/readODS")
require(readODS)
write_fods(mtcars, "mtcars.fods")
```
